v2/tools/generator/go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/devigned/tab v0.1.1
 	github.com/go-openapi/jsonpointer v0.19.5
 	github.com/go-openapi/spec v0.20.4
-	github.com/gobuffalo/flect v0.2.3
+	github.com/gobuffalo/flect v1.0.0
```

Comment:
Ooh that's a big version bump!

Reply:
Yeah, the module was already quite stable but we just kept it as v0, but I wanted to make it v1 since we are planning the next version, v2. Also, there was a heavy change in supporting irregular plurals, but the major version number doesn't mean a big change though.